docs/reference.md

```diff
@@ -201,13 +198,50 @@ generalized. Tests exercising undefined behavior do not contribute towards
 the understanding of the op's behavior.
 
 **(G6) While writing tests for floating-point type, to what precision the
-results need to be specified in llvm lit checks?**
+expected result need to be specified in checks?**
+
+For elementary operations (addition, subtraction, multiplication, division, and
+square), an implementation following IEEE specification is expected to provide a
+rounded result within 0.5 ULP of the mathematically exact result. That said, we
+can safely imagine the expected result coming out of these operations to be
```

Comment:
Nit with 80 line character:
```
can safely imagine the expected result coming out of these operations to be at
most 1 ULP apart. However, this may not work for transcendental functions
```